spec/requests/learning_units_controller_spec.rb

```diff
@@ -4,21 +4,48 @@
   describe 'GET /show' do
     let(:user) { create(:user) }
     let(:learning_unit) { create(:learning_unit) }
+    let(:curriculum) { create(:curriculum) }
 
     before do
       sign_in user
     end
 
-    context 'when accesing to the learning unit page' do
-      def perform
+    def perform(curriculum_id: nil)
```

Comment:
It is a good practice to define a perform function with parameters? I looked for and didn't found anything.